packages/google_sign_in/google_sign_in/ios/Classes/FLTGoogleSignInPlugin.m

```diff
@@ -76,7 +76,8 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
       NSString *path = [[NSBundle mainBundle] pathForResource:@"GoogleService-Info"
                                                        ofType:@"plist"];
       if (path) {
-        NSMutableDictionary *plist = [[NSMutableDictionary alloc] initWithContentsOfFile:path];
+        NSMutableDictionary<NSString *, NSString *> *plist =
+            [[NSMutableDictionary alloc] initWithContentsOfFile:path];
         BOOL hasDynamicClientId =
             [[call.arguments valueForKey:@"clientId"] isKindOfClass:[NSString class]];
```

Comment:
Maybe use that new-fangled `[]` access here while you are modernizing :)

(We could consider doing a sweep of the repo for old-style dictionary and array reads and writes and updating them, like I did for literals.)

Reply:
Done.